src/SFA.DAS.EmployerAccounts.Web/ViewModels/AccountDashboardViewModel.cs

```diff
@@ -26,9 +26,13 @@ public class AccountDashboardViewModel
         public bool AgreementsToSign { get; set; }
         public int SignedAgreementCount { get; set; }
         public List<PendingAgreementsViewModel> PendingAgreements { get; set; }
-        public AccountDto AccountViewModel { get; set; }
+        public bool ApprenticeshipAdded { get; set; }
+        public bool ShowSearchBar { get; set; }
+        public bool ShowMostActiveLinks { get; set; }
+        public EAS.Portal.Client.Types.Account AccountViewModel { get; set; }
         public Guid? RecentlyAddedReservationId { get; set; }
-        public IReservedFundingDto ReservedFundingToShow { get; set; }
-        public string ReservedFundingToShowLegalEntityName { get; set; }
+        public Reservation ReservedFundingToShow => AccountViewModel?.Organisations?.SelectMany(org => org.Reservations).FirstOrDefault(rf => rf.Id == RecentlyAddedReservationId) ?? AccountViewModel?.Organisations?.SelectMany(org => org.Reservations)?.LastOrDefault();
+        public string ReservedFundingOrgName => AccountViewModel?.Organisations?.Where(org => org.Reservations.Contains(ReservedFundingToShow)).Select(org => org.Name).FirstOrDefault();
+        public bool ShowReservations => AccountViewModel?.Organisations?.FirstOrDefault().Reservations?.Count > 0;
```

Comment:
If there are no Organisations in the collection, this will throw a `NullReferenceException`.

Reply:
It shouldn't do, everything should default to null/false. I've had no issues running locally